docs/contributing/documentation/overview.mdx

```diff
@@ -0,0 +1,71 @@
+---
+title: "Documentation"
+sidebarTitle: "Overview"
+description: "Like every other part of Mage, our documentation is open-source. Read on to learn how to make edits or even write entirely new docs for Mage!"
+---
+
+<div align="center">
+<img src="https://github.com/mage-ai/mage-ai/assets/59450879/769a5faf-37ea-453c-8bd6-83270366fe00" width="65%"/>
+</div>
+
+## Overview
+
+Mage uses [Mintlify](mintlify.com/) as our documentation framework. Mintlify provides hosting and deployment for our docs, which makes editing quite simple. In this guide, we'll discuss different "levels" of contributing to our docs, starting from simplest to most advanced.
+
+All of our docs are written in plain text as Markdown files. If you're not familiar with Markdown, check out [this guide](https://www.markdownguide.org/basic-syntax/) to get started. You can take a look at the structure and contents of our docs by navigating this site and checking out the [docs directory](https://github.com/mage-ai/mage-ai) on GitHub.
+
+## Level 1: Suggesting Edits 🦸🏻‍♂️
+
+The very simplest way to contribute to the Mage docs is directly in the browser, using GitHub's file editing feature. This is a great way to suggest small edits to existing docs, but it's not the best way to add entirely new docs or complex formatting, images, etc.
+
+To make edits, simply click on the "Suggest Edit" button at the bottom of every page. This will open a new tab with the file you're editing. Make your changes, then click "Commit changes." This will open a new pull request with your changes. Be sure to add appropriate details to your pull request & tag `@wangxiaoyou1993` or `@mattppal`. Once the pull request is merged, your changes will be live on the site!
+
+<Frame>
+  <img src="https://github.com/mage-ai/assets/blob/main/contributing-docs.png?raw=True" />
+</Frame>
+
+## Level 2: Add complex components 💪
+
+### Images
+
+To add images to the Mage docs, you'll need to contact a team member or host the images in GitHub. Our assets currently live in a private repo to keep the docs repo lightweight.
```

Comment:
the assets repo is actually public, but not sure what the process is for contributing to it

Reply:
Hm yeah maybe we keep this process for now and cross that bridge when we get there.